Updates in DIY Kit - Added using the kit - Explorer Pack and Beast Pack (#85)
upsidedownlabs/docs.upsidedownlabs.tech: kits/npg-lite-kits/npg-lite-beast/index.rst

This makes VibZ+ ideal for experiments where higher-resolution input or greater spatial coverage is essential, 
such as multi-channel EEG, EMG, or complex biofeedback applications.

Using the Kit
********************

This kit is designed for advanced biosignal research, enabling users to quickly start recording and visualizing
multi-channel data for high-resolution HCI/BCI experiments.
The VibZ+ Playmate significantly enhances acquisition by including an additional 3-channel BioAmp, allowing 
simultaneous recording of up to six biosignal channels. It retains the vibration motor, buzzer, and QWIIC port, 
making it ideal for multi-channel EEG, EMG, or complex biofeedback applications.

Step 1: Skin Preparation
=========================

Apply Nuprep skin preparation Gel on the target areas where the
Electrodes will be placed → Rub in circular motion → Clean with an alcohol swab or wet wipe.

For more information, please check out detailed step by step :ref:`Skin Preparation Guide <skin-preparation>`.

Step 2: Connect BioAmp Cables
===============================

Using VibZ+ you can record 3-channel biopotential signals, connect the jumper

> wires according to the number of channels required:

Reviewer: ```suggestion

Using VibZ+, you can record up to 6-channel biopotential signals, just connect the jumper
wires according to the number of channels required:
```

+---------------+-------------+------------------------+
| **Channel 5** | **A5P**     | +ve of Channel 5       |
+---------------+-------------+------------------------+
|               | **A5N**     | -ve of Channel 5       |
+---------------+-------------+------------------------+
|               | **REF**     | Reference cable to REF |
+---------------+-------------+------------------------+
|               | **CN/ CNB** | Common Negative        |
+---------------+-------------+------------------------+

Using the Common Negative (CN)
---------------------------------

The Common Negative (CN) is an essential feature for multi-channel recordings like EEG, as it simplifies
electrode placement and wiring. This new playmate replaces the previous solder jumpers with a convenient 
slide switch for instant configuration.

*This is the biggest change: No soldering is required. Flipping the switch physically connects all negative inputs to the common line.*

To configure the board to use the Common Negative, follow these steps:

1. **Connect the Shared Electrode:** Connect your single, common negative electrode jumper cable to the dedicated CN pin header.

> 2. **Locate the Switch:** Find the ``COMMON NEGATIVE`` slide switch on the top of the board. The silkscreen text ``FLIP THE SWCH FOR COMMON NEGATIVE`` points directly to it.

Reviewer: ```suggestion
2. **Locate the Switch:** Find the ``COMMON NEGATIVE`` slide switchs on the bottom of the board. The silkscreen text says ``FLIP THE SWITCH FOR COMMON NEGATIVE " and points directly to it.
```

.. figure:: media/npg-lite-EOG-Vertical-vibz+.*
    :align: center
    :alt: Connection for EOG Vertical
    
    Connection for EOG Vertical

EEG (Electroencephalogram)
----------------------------

- **1-CH Setup**: Place +ve cable of Channel 0 on the targeted area, REF and -ve cable on the bony part at the back of each earlobe. (refer to the `International 10-20 system for recording EEG <https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)>`_)

- **2-CH Setup**: Place +ve cables of Channel 0 and 1 on the targeted area (refer to the `International 10-20 system for recording EEG <https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)>`_), 1 REF and 1 CN (Common Negative) cable on the bony parts at the back of each earlobe.

- **3-CH Setup**:  Place +ve cables of Channel 0, 1 and 2 on the targeted area (refer to the `International 10-20 system for recording EEG <https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)>`_), 1 REF and 1 CN (Common Negative) cable on the bony part at the back of each earlobe.

> - **General Multi-Channel Setup:** For any multi-channel recording (2-CH or more), place the +ve cable for each respective channel on the targeted areas, then use only one REF cable and one CN (Common Negative) cable for all channels, placing them on your chosen reference points. (refer to the `International 10-20 system for recording EEG <https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)>`_)

Reviewer: ```suggestion

- **2-CH Setup**: Place +ve cables of Channel 0 and 1 on the targeted area (refer to the `International 10-20 system for recording EEG <https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)>`_), 1 REF and 1 CN (Common Negative) cable on the bony parts at the back of each earlobe. Make sure the common negative switch for the respective channels is flipped accordingly, as per the information provided above.

- **3-CH Setup**:  Place +ve cables of Channel 0, 1 and 2 on the targeted area (refer to the `International 10-20 system for recording EEG <https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)>`_), 1 REF and 1 CN (Common Negative) cable on the bony part at the back of each earlobe. Make sure the common negative switch for the respective channels is flipped accordingly, as per the information provided above.

- **General Multi-Channel Setup:** For any multi-channel recording (2-CH or more), place the +ve cable for each respective channel on the targeted areas, then use only one REF cable and one CN (Common Negative) cable for all channels, placing them on your chosen reference points. (refer to the `International 10-20 system for recording EEG <https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)>`_). Make sure the common negative switch for the respective channels is flipped accordingly, as per the information provided above.
```
